### DISCUSSION TITLE
attu界面的rootcoord节点显示NaN

### DISCUSSION THREAD

**Discussion by dushulin** [milvus-io/milvus#29080] [Q&A and General discussion]
![image](https://github.com/milvus-io/milvus/assets/25276301/7dcba5b2-58d1-4acf-a0aa-06b767b82568)

**Comment by shanghaikid** [comment]
what's your attu version?

**Reply by dushulin** [reply]
image: zilliz/attu:v2.2.8

**Reply by shanghaikid** [reply]
I think the issue had been fixed in the newer version of attu.
